apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md

```diff
@@ -2,6 +2,7 @@
 eleventyNavigation:
     key: Our vision
     parent: All about PIE
+site_title: "Our vision"
 title: Our vision
```

Comment:
do you think we can use one property instead? 
it seems most of the cases site_title and title are referring to the same thing.

Reply:
done!